tox.ini

```diff
@@ -82,6 +86,35 @@ commands =
     bash -c 'coverage run --source={envsitepackagesdir}/pudl -- $(which sphinx-build) -W -b html docs docs/_build/html'
     coverage xml
 
+###########################################################################
+# Continuous integration tests.
+# 
+# The following steps are run in sequence as part of the CI workflows on
+# github to run the ETL and integration tests in a modular fashion.
+###########################################################################
+[testenv:ci_ferc_to_sqlite]
+description = Run ferc_to_sqlite script in the CI environment.
+envdir = {toxworkdir}/.ci_env
+runner = ignore_env_name_mismatch
+commands =
+    bash -c 'coverage run --append src/pudl/ferc_to_sqlite/cli.py {gcs_cache} src/pudl/package_data/settings/etl_fast.yml'
+
+[testenv:ci_pudl_etl]
+description = Run the ETL script in the CI environment.
+envdir = {toxworkdir}/.ci_env
+runner = ignore_env_name_mismatch
+commands =
+    bash -c 'alembic upgrade head'
+    bash -c 'coverage run --append src/pudl/etl/cli.py {gcs_cache} src/pudl/package_data/settings/etl_fast.yml'
```

Comment:
When I tried running the ETL with coverage like this, it fully utilized my CPUs, but I only got ~35% coverage, which seemed to reflect what you'd get from just importing PUDL, with all of the code that was actually run by the Dagster processes not showing up.

Reply:
Perhaps we need to run codecov in a multi process friendly way.

Reply:
I tried it with `--parallel-mode` and it didn't change the behavior. I don't really understand the difference between subprocesses that the parent process can keep track of, or entirely separate processes that have been spawned some other way?